vault/blog.2021.2021-09-27-markdown-on-mobile.md

```diff
@@ -1,66 +1,78 @@
 ---
 id: fDCVPEo3guCFWPdxokXHU
-title: 'Mobile apps for notes in Markdown: A comparison'
+title: 'Best Mobile Note-Taking Apps for Markdown'
 desc: ''
-updated: 1632684719327
+updated: 1634758861309
 created: 1632684719327
-date: '2021-09-27'
+date: '2021-10-20'
 image: "https://org-dendron-public-assets.s3.amazonaws.com/images/blog-mobile-editor-header.png"
-excerpt: "A stroll through mobile apps that can take notes in Markdown and make use of git"
+excerpt: "A stroll through mobile apps that can take Markdown-style notes"
 author: "Derek Ardolf"
 ---
 
 ![Photo of a phone ontop of a notebook](https://org-dendron-public-assets.s3.amazonaws.com/images/blog-mobile-editor-header.png)
 
-On the desktop, markdown-based notes can be taken in any plain-text editor of choice. Dendron sits ontop of VSCode/VSCodium. Though, what happens when you walk out the door? I know you look both ways before crossing the street with your open laptop, but it turns out that your phone can work with markdown, too!
+On the desktop, markdown-based notes can be taken in any plain-text editor of choice. If you're using [Dendron](https://www.dendron.so/), you can use VSCode/VSCodium as well as a suite of [powerful commands](https://wiki.dendron.so/notes/eea2b078-1acc-4071-a14e-18299fc28f47.html) to manage your notes. Though, what happens when you walk out the door? It turns out that your phone can work with markdown, too!
 
-Will these apps sway users to adopt them, and end up in news articles as the cause of distracted drivers? Please stop driving. Pull over, _right now_, and continue reading from that gas station parking lot.
-
-What we're looking at today:
+What we're looking at today, with a bias toward Android-compatible tooling due to the absence of Apple devices in my life:
```

Comment:
since most of the tools your going over are cross platform, I would drop the android bias warning